build.gradle

```diff
@@ -207,8 +212,35 @@ subprojects {
         finalizedBy jacocoTestReport
     }
 
-    jacocoTestReport {
-        dependsOn test
+    task newIntegrationTests(type: Test) {
```

Comment:
Are you planning to switch this to just `integrationTests` after deprecating the old ones?

Reply:
Not really, the old integrationTest task include some acceptance test. I think that we should switch the existing integrationTest task to acceptanceTest.